File: curso python/ejemplos python/positivo_negativo_neutro.py
num=int (input('Ingrese el numero a verificar: '))
def verifi(num):
    if num>0:
        res='POSITIVO'
    if num<0:
        res = 'NEGATIVO'
    if num==0:
        res= 'NEUTRO'
    return res
print ('el numero es: ', verifi(num))

File: curso c++/ejemplos/positivo_negativo_neutro.cpp
#include <iostream>
using namespace std;
int main (){
    int num;
    cout << "ingrese el numero: ";
    cin >> num;
    if (num >0 )
        cout <<"el numero: "<< num<< " es POSITIVO";
    else
        if (num<0)
            cout << "el numero: "<<num<<" es NEGATVO";
        else
            cout << "el numero: "<<num<< " es NEUTRO";


    return 0;
}